### PR TITLE
Create components folder for reused SCSS

### PR DESCRIPTION
**Pull request by smharley**
* Add normalize-rails gem, remove normailze vendor file
* Create `components/` folder, remove `application/`
* Consolidate helpers

Most of this PR is still structural, and not actually refactoring the code, just cleaning things up

### CODE REVIEW

**Comment by smharley** [comment]
image
image

**Comment by smharley** [comment]
Is someone able to pull this branch down that has ngrok running? I want a sanity check that the repos page looks exactly the same

**Comment by thorncp** [comment]
Seems legit to me :mouse:

File: vendor/assets/stylesheets/_normalize.scss [hunk 1]
```diff
@@ -1,2 +0,0 @@
-article,aside,details,figcaption,figure,footer,header,hgroup,main,nav,section,summary{display:block}audio,canvas,video{display:inline-block}audio:not([controls]){display:none;height:0}[hidden]{display:none}html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}a:focus{outline:thin dotted}a:active,a:hover{outline:0}h1{font-size:2em;margin:.67em 0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}hr{-moz-box-sizing:content-box;box-sizing:content-box;height:0}mark{background:#ff0;color:#000}code,kbd,pre,samp{font-family:monospace, serif;font-size:1em}pre{white-space:pre-wrap}q{quotes:\201C \201D \2018 \2019}small{font-size:80%}sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}fieldset{border:1px solid silver;margin:0 2px;padding:.35em .625em .75em}button,input,select,textarea{font-family:inherit;font-size:100%;margin:0}button,input{line-height:normal}button,select{text-transform:none}button,html input[type=button],/* 1 */
```
Comment (smharley) [inline vendor/assets/stylesheets/_normalize.scss]: this looks more like a reset than normalize. could cause some issues with spacing being different than before

**Pull request merged**